snippets/es6.cson

```diff
@@ -0,0 +1,74 @@
+'.source.coffee':
+
+  'Ember Snippets for es6 syntax':
+    'prefix': 'es6--help'
+    'body':
+      """
+      ${1:
+        # snippets to generate --ES6 syntax--
+        # example: imp-ds<press tab>
+        #  ___________________________________________________________
+        # | type short (or)  long version    ->   then press <tab>    |
+        # | ----------  ->  ---------------  -> ----------------------|
+        # |  imp-ds     ->   import-ds       ->  `import DS...      ` |
+        # |  imp-emb    ->   import-ember    ->  `import Ember...   ` |
+        # |  exp-def    ->   export-default  ->  `export default... ` |
+        # |  es6        ->   es6             ->  - full set -         |
+        # |___________________________________________________________|
+        }
+      """
+
+  'ES6 import Ember':
+    'prefix': 'import-ember'
```

Comment:
keep in mind that the package is written to primarily work with the long version.
it does not force anyone to use or memorize shorthands.
They are just sitting there for seriously lazy people like me to pickup. ;)

/even the zen-coding (or emmet) has such ugly optional shorthands in it./